tests/zero_code_change/tensorflow2_integration_tests.py

```diff
@@ -157,26 +157,18 @@ def helper_test_keras_v2_gradienttape(script_mode: bool = False, json_file_conte
             else:
                 hook = smd.KerasHook.create_from_json_file()
             opt = hook.wrap_optimizer(opt)
+            tape = tf.GradientTape(persistent=True)
+            tape = hook.wrap_tape(tape)
             for epoch in range(n_epochs):
                 for data, labels in dataset:
                     dataset_labels = labels
                     labels = tf.one_hot(labels, depth=10)
-                    with tf.GradientTape(persistent=True) as tape:
-                        # call this API before a forward pass to prepare collections
-                        # and initialize writers
-                        hook.forward_pre_hook(tape)
+                    with tape:
```

Comment:
Nit: Encouraging `with hook.wrap_tape(tf.GradientTape())` rather than declaring variables in advance is even cleaner, then the user only has a one-line change to make.

Reply:
done in the latest commit